codegen/templates.go

```diff
@@ -69,61 +69,65 @@ type templateHandler struct {
 }
 
 // Write takes one endpoint and uses a template to generate text
-// for it. This template is written to the given writer.
-func (h *templateHandler) Write(wr io.Writer, tmplType templateType, endpoint string, endpointInfo *framework.OASPathItem) error {
+// for it. This template is written to the given writer. It's exported
+// because it's the only method intended to be called by external callers.
+func (h *templateHandler) Write(wr io.Writer, endpoint string, endpointInfo *framework.OASPathItem, addedInfo *additionalInfo) error {
```

Comment:
Same question here re: pointer receiver.

Reply:
Same comments as above.

Reply:
Marking as resolved to consolidate the conversation to https://github.com/terraform-providers/terraform-provider-vault/pull/749#discussion_r428298305.